daemon/container_unix.go

```diff
@@ -976,9 +987,9 @@ func (container *Container) connectToNetwork(idOrName string, updateSettings boo
 		if err != nil {
 			return err
 		}
-	}
 
-	container.updateSandboxNetworkSettings(sb)
+		container.updateSandboxNetworkSettings(sb)
```

Comment:
@mrjana Sorry I'm not sure I understand. I guess I don't see how that answers my question: why don't we need to call `updateSandboxNetworkSettings` with the `sb` returned from the walk, when currently in master that's what we do?